interchain/umee-1/umeeosmo.json

```diff
@@ -0,0 +1,20 @@
+{
+    "src": {
+      "chain-id": "umee-1",
+      "client-id": "07-tendermint-6",
+      "connection-id": "connection-0",
+      "channel-id": "channel-0",
+      "port-id": "transfer",
+      "order": "unordered",
+      "version": "ics20-1"
+    },
+    "dst": {
+      "chain-id": "cosmoshub-4",
+      "client-id": "07-tendermint-1805",
```

Comment:
Everything looks good except this light client. 

Can you verify `07-tendermint-1805`

Reply:
TY! 🙏